content/en/tutorials/deploying-ecs-image-eks-localstack.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Deploying an ECS Image to EKS with LocalStack"
+linkTitle: "Deploying an ECS Image to EKS with LocalStack"
+weight: 7
+description: >
+  This tutorial showcases how to deploy an ECS image to EKS locally with LocalStack.
+cascade:
+  type: docs
+---
+
+In this tutorial we will, by the use of an example, explore the usage of ECR images inside EKS.
```

Comment:
@HarshCasper Given that "tutorials" are targeting beginners: would it make sense to also include a short, general description about ECR and EKS (what it stands for, what it is generally used for)?

Reply:
Hi @steffyP — This is just for placeholder. I intend to re-work on them later in point to ensure tutorials are to the point.